packages/shared/src/schema/expoJudgingVote/post.ts

```diff
@@ -0,0 +1,6 @@
+import z from 'zod';
+
+export const post = z.object({
+  currentProjectChosen: z.boolean(),
+  expoJudgingSessionId: z.string().regex(/[0-9]/g),
```

Comment:
Let's leave this off; it's safe to query by whatever ID is provided so let's keep it simple in case we ever decide to swap ID to UUIDs or something
```suggestion
  expoJudgingSessionId: z.string(),
```